python_modules/dagstermill/dagstermill/__init__.py

```diff
@@ -29,6 +30,7 @@
     check,
     types,
 )
+from dagster.core.types.marshal import PickleSerializationStrategy
```

Comment:
I'd kind of like to see this grouped with the other imports from `dagster.core.types.marshal`